packages/lwc/types.d.ts

```diff
@@ -178,4 +180,24 @@ declare module 'lwc' {
      * @param config configuration object for the accessor
      */
     export function wire(getType: (config?: any) => any, config?: any): PropertyDecorator;
+
+    type ConfigValue = Record<string, any>;
```

Comment:
what do you think about prefixing these (`ConfigValue`, `ContextValue` and `DataCallback`) with `Wire`, in order for them to match https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-core/src/framework/main.ts#L43 ?